scripts/editor-support.js

```diff
@@ -10,11 +11,27 @@ import {
 import { decorateRichtext } from './editor-support-rte.js';
 import { decorateMain } from './scripts.js';
 
+/**
+ *
+ * @param {HTMLElement} block
+ * Use this function to trigger a mutation for the UI editor overlay when you
+ * have a scrollable block
+ */
+function createMutation(block) {
```

Comment:
This is a trick used to "trick" UE (and its mutation observer) into recalculating the layout, because scroll on elements doesn't trigger the observer. Without it, when you click on an editable area in UE, the borders (the blue selector) would not match the actual content as it would match the previous elements which may be in different positions on the screen.